src/main/java/io/codelex/classesandobjects/practice/exercise_2/Car.java

```diff
@@ -0,0 +1,46 @@
+package io.codelex.classesandobjects.practice.exercise_2;
+
+public class Car {
+    private double startKilometers;
+    private double endKilometers;
+    private double liters;
+
+
+    public Car(double startOdo, double endingOdo, double liters) {
+        startKilometers = startOdo;
+        endKilometers = endingOdo;
+        this.liters = liters;
+    }
+
+    public Car(double startOdo) {
+        startKilometers = startOdo;
+    }
+
+    public double[] calculateConsumption() {
+        double litersPer100km = liters / (endKilometers - startKilometers) * 100;
+        double kilometersPerLitre = 100 / litersPer100km;
+        double[] ar = new double[2];
+        ar[0] = litersPer100km;
+        ar[1] = kilometersPerLitre;
```

Comment:
neredzu šim īsti jēgu, nekur arī nepielieto? ja vēlies sadalīt labāk tad izveidot divas metodes vai arī jaunu objektu

Reply:
"ReadMe" failā bija noradīts, ka main metodē jāizdrukā : "Kilometers per liter are:... ". Tur arī izmantoju ar[1], kur aprēķināts km per liter, savukārt gasHog funkcijā jāsalīdzina l/km. Vai varbūt uzdevuma nosacījumos bija sajaukts vietām un nevajadzēja nekādus "kilometers per liter", bet gan l/km? Tas atvieglotu dzīvi. :D

Reply:
ļoti iespējams jā, but just in general, nav diezko labs atgriešanas tips, thats about it

Reply:
Izveidoju atsevišķu metodi.